### PR TITLE
4.20 release pre-PR

### PR DESCRIPTION
**Pull request by vanhauser-thc**

### CODE REVIEW

**Comment by domenukk** [comment]
Just in time for legalization in Germany

File: docs/INSTALL.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ If you want to build AFL++ yourself, you have many options. The easiest choice
 is to build and install everything:
 
 NOTE: depending on your Debian/Ubuntu/Kali/... release, replace `-14` with
-whatever llvm version is available. We recommend llvm 13, 14, 15 or 16.
+whatever llvm version is available. We recommend llvm 13 or newer.
```
Comment (domenukk) [inline docs/INSTALL.md:24]: Shouldn't we recommend the latest stable version for performance/optimization reasons?

File: include/t1ha0_ia32aes_b.h [hunk 2]
```diff
@@ -0,0 +1,183 @@
+/*
```
Comment (domenukk) [inline include/t1ha0_ia32aes_b.h:1]: Stupid question, looks like this is the zLIB license, compatible with GPL, but so far we have apache everywhere else.. could that pose a problem for users?

Comment (vanhauser-thc) [reply include/t1ha0_ia32aes_b.h:1]: Very good question. Will check it out

Comment (vanhauser-thc) [reply include/t1ha0_ia32aes_b.h:1]: https://www.apache.org/legal/resolved.html states it’s compatible

**Pull request merged**